lib/src/tinycolor.dart

```diff
@@ -14,6 +14,7 @@ class TinyColor {
 
   TinyColor(this.originalColor) : _color = Color(originalColor.value);
 
+  @Deprecated('Use TinyColor(Color.fromARGB(a, r, g, b)) instead.')
```

Comment:
Maybe add note with version number indicating form which version on this was deprecated. I.e. 'This was deprecated in v2.0.1" or similar

Reply:
This PR is planned for v2.1.0, not v2.0.1. See milestone set for this.

Since there are few BREAKING CHANGES in v2.1.0, it is going to be documented in the changelog.

Reply:
Ok, add v2.1.0 then in the deprecated description

Reply:
Per [Dart docs](https://api.dart.dev/stable/2.16.1/dart-core/Deprecated/Deprecated.html), it says:

> The [message](https://api.dart.dev/stable/2.16.1/dart-core/Deprecated/message.html) argument should be readable by programmers, and should state an alternative feature (if available) as well as when an annotated feature is expected to be removed.

Adding when it was deprecated serves no point. This info can always found at changelog, and does not help when to migrate.

Reply:
At least in flutter it is common to do so: https://github.com/flutter/flutter/wiki/Tree-hygiene#deprecations
But whatever, then just skip it...

Reply:
> At least in flutter it is common to do so: https://github.com/flutter/flutter/wiki/Tree-hygiene#deprecations
> But whatever, then just skip it...

Upon checking `dart.dev` & `flutter.dev` managed repos, they don’t specific since as well.

- https://github.com/flutter/plugins/commit/3990aaf18eb81262efac6a6477d9f3536e69b44b#diff-4cc76aa8c6f7d7f57f0e0ff337aa4e1e163bb939ddd4e3a33b8afb2168c9236aR85
- https://github.com/flutter/plugins/commit/936257f69eb2a6978b146e6c32cb453e75d6e923#diff-bbee1da5da193a072dca86d16467e75802852826099a7b0db2e6c497c71a3f31L62
- https://github.com/dart-lang/collection/commit/db9f04de6b6fea3949c6b77df6f7367c9245be92#diff-9779fe1a473c63bae1ece37553462802715356402709ecaaa3df19cbf1728efcR14

Since Dart & Flutter are made by Google, I can assume that wiki page is for Flutter main repo only.

Reply:
ok, go ahead without it. no problem